src/components/TrialBanner.js

```diff
@@ -63,6 +63,7 @@ export const TrialBanner = Utils.connectStore(authStore, 'authState')(class Tria
     const { finalizeTrial } = this.state
     const { trialState } = profile
     const removeBanner = localStorage.getItem('removeBanner')
+    if (trialState === 'Enabled') return null // Disable new sign-ups
```

Comment:
TOL: This effectively removes the Enabled message, which is the only one with `isExternal: false`. That means that the last `freeCreditsActive.set(true)` is unreachable, so we could prune that state and the whole `FreeCreditsModal` if we want.